src/repart/repart.c

```diff
@@ -7486,7 +7506,29 @@ static int context_fstab(Context *context) {
                 }
         }
 
-        r = flink_tmpfile(f, t, path, 0);
+        switch (arg_append_fstab) {
+        case APPEND_AUTO:
+                r = read_full_file(path, &c, &l);
```

Comment:
please output an error for unexpected errors, and fail. for expected errors (i.e. ENOENT) a debug msg might be nice too